viewer/packages/cad-geometry-loaders/src/sector/culling/takensectors/TakenV8SectorMap.ts

```diff
@@ -66,13 +66,9 @@ export class TakenV8SectorMap extends TakenSectorMapBase {
     sectorTree!.markSectorDetailed(sectorId, priority);
   }
 
-  isWithinBudget(budget: CadModelSectorBudget): boolean {
+  isWithinBudget(budget: CadModelBudget): boolean {
     const currentCost = this.totalCost;
-    return (
-      currentCost.downloadSize < budget.geometryDownloadSizeBytes &&
-      currentCost.drawCalls < budget.maximumNumberOfDrawCalls &&
-      currentCost.renderCost < budget.maximumRenderCost
-    );
+    return currentCost.renderCost < budget.maximumRenderCost;
```

Comment:
Just a general question - do all V8 sectors specify both estimated number of draw calls and render cost? Because this change will apply to both V8 and V9, right?

Reply:
Yes, this is in place in v8 as well.